res/values-vi/strings.xml

```diff
@@ -521,9 +521,9 @@
     <string name="Collect_10_000_snowflakes_">"Thu thập 10.000 bông tuyết."</string>
     <string name="Show_All">"Hiển thị tất cả"</string>
 
-    <string name="SET_PERMISSIONS">"GIẤY PHÉP THIẾT LẬP"</string>
+    <string name="SET_PERMISSIONS">"THÊM QUYỀN HẠN"</string>
     <string name="Start_Clan_War">"Bắt Đầu Hội Chiến"</string>
-    <string name="Min_Level">"Mức tối thiểu"</string>
+    <string name="Min_Level">"Level tối thiểu"</string>
```

Comment:
Why "Level" here if it has a set translation?

Reply:
oh,mistake.Can you edit it to “Cấp độ”?

Reply:
Thanks for your trust, but don't forget you can still make these edits and [push a new commit with them](https://youtu.be/C-gul4cHHPc). Good luck ^^

Reply:
alright,i corrected it.Thank for your tutorial!!